client/src/components/rooms/transcriptionNotifications/CallSummaryTile.tsx

```diff
@@ -0,0 +1,129 @@
+import { fetchTranscript } from '../../../utils/CallAutomationUtils';
+/**
+ * @private
+ */
+export interface CallSummaryTileProps {
+  pageStyle?: IStyle;
+  serverCallId?: string;
+  summary?: string;
+  summarizationStatus: 'InProgress' | 'Complete' | 'None';
+}
+
+/**
+ * Generic page with a title and more details text for serving up a notice to the user.
+ *
+ * @private
+ */
+export const CallSummaryTile = (props: CallSummaryTileProps): JSX.Element => {
+  const { serverCallId, summarizationStatus, summary } = props;
+
+  const theme = useTheme();
+
+  return (
+    <Stack
+      style={{ width: 'auto', height: 'auto' }}
+      verticalFill
+      verticalAlign="center"
+      horizontalAlign="center"
+      aria-atomic
+    >
+      <Stack className={mergeStyles(containerStyle)} tokens={containerItemGap}>
+        <Stack tokens={containerItemGap}>
+          {summarizationStatus === 'InProgress' && (
+            <Spinner styles={{ root: { marginTop: '2rem' } }} label="Summarizing conversation..." />
+          )}
+          {summarizationStatus !== 'InProgress' && (
+            <Stack
+              styles={{
+                root: {
+                  width: '100%',
+                  borderRadius: theme.effects.roundedCorner4,
+                  boxShadow: theme.effects.elevation8
+                }
+              }}
+            >
+              <Stack styles={{ root: { padding: '1rem' } }}>
+                <Text styles={{ root: { fontWeight: 600 } }} variant="large">
+                  {summary ? 'Meeting Summary' : 'Meeting Transcript'}
+                </Text>
+              </Stack>
+
+              <Stack styles={{ root: { padding: '0 1rem' } }}>
+                <Stack
+                  styles={{
+                    root: {
+                      width: '100%',
+                      height: '1px',
+                      backgroundColor: theme.palette.neutralLight
+                    }
+                  }}
+                />
+              </Stack>
+
+              <Stack tokens={{ childrenGap: '0.5rem', padding: '1rem' }}>
+                <Stack horizontal tokens={{ childrenGap: '0.5rem' }} verticalAlign="center">
+                  <Icon style={{ color: theme.palette.neutralSecondary }} iconName={'Info'}></Icon>
+                  <Text styles={{ root: { fontWeight: 400, color: theme.palette.neutralSecondary } }}>
+                    {'Generated by AI. Check for accuracy.'}
+                  </Text>
+                </Stack>
+                {summary && <Text styles={{ root: { fontWeight: 400 } }}>{summary}</Text>}
+                <DefaultButton
+                  text="Transcript"
+                  onClick={async () => {
+                    if (summary && serverCallId) {
+                      const transcript = await fetchTranscript(serverCallId);
+                      if (!transcript) {
+                        console.error('Failed to fetch transcript');
```

Comment:
Is there is subcode/code errors that are surfaced from the fetchTranscript?

Reply:
yes from the server response. this is just a more readable error.